aten/src/ATen/native/sparse/cuda/SparseCsrTensorMath.cu

```diff
@@ -100,6 +105,35 @@ void convert_indices_from_csr_to_coo_cuda(const Tensor& indices, const Tensor& c
   C10_CUDA_KERNEL_LAUNCH_CHECK();
 }
 
+template <typename scalar_t, typename value_t>
```

Comment:
Use `using value_t = typename c10::scalar_value_type<scalar_t>::type;` inside the function.
```suggestion
template <typename scalar_t>
```